.ci/docker/Dockerfile

```diff
@@ -1,45 +1,60 @@
-FROM ubuntu:trusty
+FROM ubuntu:bionic
 
-ARG PARALLEL=-j8
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8 PATH=/opt/bin:/opt/ghc/bin:/opt/cabal/bin:$PATH
 
-ENV DEBIAN_FRONTEND noninteractive
-ENV PATH /opt/ghc/bin:/opt/cabal/bin:$PATH
-
-ARG DEPS_GHC="pkg-config libgmp-dev curl"
-ARG DEPS_CLASH="libtinfo-dev"
-ARG DEPS_CLASH_COSIM="g++-multilib make"
-ARG DEPS_IVERILOG_BUILD="autoconf bison flex gperf"
```

Comment:
I think it's useful to keep these variables as documentation why these package are installed.